pkg/apis/rollouts/v1alpha1/types.go

```diff
@@ -413,6 +413,8 @@ type RolloutExperimentTemplate struct {
 	// use the same selector as the Rollout
 	// +optional
 	Selector *metav1.LabelSelector `json:"selector,omitempty" protobuf:"bytes,5,opt,name=selector"`
+	// Weight sets the percentage of traffic the template's replicas should receive
+	Weight *int32 `json:"weight,omitempty" protobuf:"varint,6,opt,name=weight"`
```

Comment:
To keep things cleaner, can we move the introduction of this field with the separate PR which integrates experiments with traffic routing weights?